### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wora**
- Added "-f bare" for feeding fetch response to info.
- Added gcurl alias example.
- Simplified some description.
- Added note for credential cache.

### CODE REVIEW

**Comment by wora** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -65,24 +69,32 @@ the following command uses the BigQuery API to list all projects.
 $ curl -H "$(oauth2l header bigquery)" 'https://www.googleapis.com/bigquery/v2/projects'
 ```
 
+If you need to call Google APIs frequently using the `header` command, you
+might define a shell alias for it, for example:
+
+```
+$ alias gcurl='curl -H "$(oauth2l header cloud-platform) -H "Content-Type: application/json" '
```
Comment (craigcitro) [inline README.md]: Need another `"` after `)`

**Comment by craigcitro** [comment]
LGTM, just one small nit in an alias.

**Comment by wora** [comment]
Fixed.

**Pull request merged**